### PR TITLE
[#877] Apply UI linting to projects module

### PR DESCRIPTION
**Pull request by aamirxshaikh**
### Purpose

Applying UI linting to projects module. Removing projects module from ui/.eslintignore and ui/.prettierignore and fixing formatting.

### CODE REVIEW

File: ui/projects/streampipes/platform-services/src/lib/apis/datalake-rest.service.ts [hunk 1]
```diff
@@ -19,190 +19,243 @@
...
+    ): Observable<SpQueryResult> {
+        const columns = queryParams.columns;
+        if (columns === '') {
+            const emptyQueryResult = new SpQueryResult();
+            emptyQueryResult.total = 0;
+            return of(emptyQueryResult);
+        } else {
+            const url = this.dataLakeUrl + '/measurements/' + index;
+            const headers = ignoreLoadingBar ? { ignoreLoadingBar: '' } : {};
+            // @ts-ignore
```
Comment (tenthe) [inline ui/projects/streampipes/platform-services/src/lib/apis/datalake-rest.service.ts:88]: I think this is the reason why the PR validation failed.
Can you try to replace it with:
```
 return this.http.get<SpQueryResult>(url, {
   params: queryParams as unknown as HttpParams,
   headers,
});
```

**Review by tenthe — CHANGES_REQUESTED** [review]

**Comment by tenthe** [comment]
Hi @aamirxshaikh thanks for providing the PR.
I tried it locally and added a comment with a fix that worked for me. Please try it out and update the PR.

**Comment by aamirxshaikh** [comment]
Thanks @tenthe I have updated the PR.

**Comment by bossenti** [comment]
@aamirxshaikh thanks for your continuous contributions 🙏🏼

**Review by tenthe — APPROVED** [review]
Thanks a lot for the contribution and updating the PR 💪

**Comment by aamirxshaikh** [comment]
Thanks @bossenti and @tenthe for helping me with fixes and documentation for the contributions 🙏🏼, I hope to contribute more 🚀

**Pull request merged**